src/main/java/org/prebid/server/bidder/orbidder/OrbidderBidder.java

```diff
@@ -120,4 +123,34 @@ private BidResponse decodeBodyToBidResponse(BidderCall<BidRequest> httpCall) {
             throw new PreBidException(e.getMessage(), e);
         }
     }
+
+    private BidderBid toBidderBid(Bid bid, String cur, List<BidderError> errors) {
+        final BidType bidType;
+        try {
+            bidType = getBidTypeFromMtype(bid.getMtype());
+        } catch (PreBidException e) {
+            errors.add(BidderError.badServerResponse(e.getMessage()));
+            return null;
+        }
+
+        return BidderBid.of(bid, bidType, cur);
+    }
+
+    private static BidType getBidTypeFromMtype(Integer mType) {
+        switch (mType) {
+            case 1 -> {
+                return BidType.banner;
+            }
+            case 2 -> {
+                return BidType.video;
+            }
+            case 3 -> {
+                return BidType.audio;
+            }
+            case 4 -> {
+                return BidType.xNative;
+            }
+            default -> throw new PreBidException("Unsupported mType " + mType);
```

Comment:
Can be simplified using `switch` as expression(Since java 17).
```java
return  switch(mType) {
      case 1 -> BidType.banner;
      ...
      default -> throw new PreBidException("Unsuppoprted mType " + mType);
}

Reply:
@And1sS : I changed to switch statement according to your input, thanks for the explanation